### PR TITLE
Update control-usb-devices-using-intune.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the Navigation

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4791

File: microsoft-365/security/defender-endpoint/control-usb-devices-using-intune.md [hunk 1]
```diff
@@ -166,9 +166,9 @@ To prevent installation of particular classes of devices:
 
 ### Block installation and usage of removable storage
 
-1. Sign in to the [Microsoft Azure portal](https://portal.azure.com/).
+1. Sign in to the [Microsoft Endpoint Manager Admin centerl](https://endpoint.microsoft.com/).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/control-usb-devices-using-intune.md]:
```suggestion
1. Sign in to the [Microsoft Endpoint Manager admin center](https://endpoint.microsoft.com/).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks.

**Pull request merged**